click_on_element_using_its_coordinates/src/main/java/com/testsigma/addons/web/DoubleClickOnCoordinates.java

```diff
@@ -0,0 +1,59 @@
+package com.testsigma.addons.web;
+
+import com.testsigma.sdk.ApplicationType;
+import com.testsigma.sdk.WebAction;
+import com.testsigma.sdk.annotation.Action;
+import com.testsigma.sdk.annotation.TestData;
+import lombok.Data;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.interactions.Actions;
+
+@Data
+@Action(actionText = "Double click on testdata using x,y coordinates",
```

Comment:
Double click on testdata using x,y coordinates
Change the NLP action text to
Double click on co-ordinates testData (format: x,y  Ex: 100,200)